nixos/tests/keymap.nix

```diff
@@ -142,6 +161,7 @@ in pkgs.lib.mapAttrs mkKeyboardTest {
       symbols.expect = [ "&" "[" "{" "}" "(" "=" "*" ")" "+" "]" "!" ];
     };
 
+    extraConfig.console.keyMap = "dvorak-programmer";
     extraConfig.services.xserver.layout = "us";
     extraConfig.services.xserver.xkbVariant = "dvp";
```

Comment:
I don't understand why i have do use `dvp` in one place and `dvorak-programmer` in the other. Where does dvp come from?

The actual file is called https://github.com/legionus/kbd/blob/master/data/keymaps/i386/dvorak/dvorak-programmer.map and in the whole repo is no mention of dvp.

Reply:
dvp is the name of the layout in [xkeyboard-config](https://github.com/freedesktop/xkeyboard-config/blob/master/symbols/us#L555) and thus has to be used to configure the layout of the xorg display server. Note that the test checks the keyboard layouts for both the linux console and the xorg server.

Reply:
thanks. i clarified it in the documentation